styles/index.css

```diff
@@ -0,0 +1,121 @@
+.main-box {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    width: 1500px;
```

Comment:
if you change the width to auto it will make your layout responsive the changing of the window size.